core-galleon-pack/src/main/resources/feature_groups/default-sockets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="default-sockets" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
This one doesn't seem right.  These sockets are associated with uses of core-service.management.management-interface.http-interface. Shouldn't these be tied more directly to the uses of those?

I see they don't belong with host-management-interfaces.xml as that's not how host.xml works. They do relate to management-interfaces.xml which is only used for standalone.

If you want these kept in a separate feature-group, e.g. for reuse in other contexts, then I think the name of the group should change. These are management-sockets.

Reply:
Renaming.